### PR TITLE
A few changes offered

### PR DESCRIPTION
**Pull request by elear**
1.  Add some descriptive text in the abstract addressing Issue #68 
2.  Define "extension point" addressing Issue #69.
3.  One editorial.

### CODE REVIEW

File: draft-iab-use-it-or-lose-it.md [hunk 1]
```diff
@@ -112,14 +116,17 @@ developed.  Experience with Internet-scale protocol deployment shows that each
...
-This document examines the specific conditions that determine whether protocol
-maintainers have the ability to design and deploy new or modified protocols.
-{{implementations}} highlights some historical examples of difficulties in
-transitions to new protocol features.  {{use-it}} argues that ossified protocols
-are more difficult to update and describes how successful protocols make
-frequent use of new extensions and code-points.  {{other}} outline several
-additional strategies that might aid in ensuring that protocol changes remain
-possible over time.
+The means by which to communicate a change or new capability to a
+protocol is called an extension point.  This document examines the
```
Comment (martinthomson) [inline draft-iab-use-it-or-lose-it.md:120]: I might have said instead:

> An extension point is a mechanism that allows a protocol to be changed or enhanced.

File: draft-iab-use-it-or-lose-it.md [hunk 2]
```diff
@@ -91,17 +91,21 @@ informative:
 
 --- abstract
 
-The ability to change protocols depends on exercising the extension and version
-negotiation mechanisms that support change.  Protocols that don't use these
-mechanisms can find it difficult and costly to deploy changes.
+The ability to change protocols depends on exercising the extension
+and version negotiation mechanisms that support change.
+Implementations that don't use these mechanisms on a regular basis can
```
Comment (martinthomson) [inline draft-iab-use-it-or-lose-it.md:96]: I see what you did here, maybe shorter:

> This document explores how regular use of new protocol features can ensure that it remains possible to deploy changes to a protocol.  Examples are given where lack of use caused changes to be more difficult or costly.

File: draft-iab-use-it-or-lose-it.md [hunk 3]
```diff
@@ -91,17 +91,21 @@ informative:
...
+find it difficult and costly to deploy changes.  This memo provides
+suggestions to maintain the value of evolutionary mechanisms.  The
+focus is primarily the peer-to-peer application layer.  Some lessons
+may be drawn elsewhere.
```
Comment (martinthomson) [inline draft-iab-use-it-or-lose-it.md:100]: I don't think that the abstract needs the caveat/disclaimer.  Keep things concise.

Comment (elear) [reply draft-iab-use-it-or-lose-it.md:100]: ok

**Pull request merged**